docs/contributing/feature-stages.md

```diff
@@ -9,37 +9,37 @@ If you encounter an issue with any Coder feature, please submit a
 
 ## Early access features
 
+- **Stable**: No
+- **Production-ready**: No
+- **Support**: GitHub issues
```

Comment:
Can you also mention or "via design partnerships. [Contact your account team](https://coder.com/contact)" or similar. Or do you think this would be implied?

Reply:
yeah. i think that's great

Reply:
much better